core-v1/src/main/scala/engine/StandardEngine.scala

```diff
@@ -311,6 +311,13 @@ object StandardEngine {
       ExprResult.Sorted(expr, finalState)
     }
 
+    // TODO: This requires an Arrow over the function type, which will probably require refactoring this trait
+    override def visitToHList[I, L <: HList : OP](
+      expr: Expr.ToHList[I, L, OP],
+    )(implicit
+      toHL: ConvertToHList[L],
+    ): PO <:< I => ExprResult[PO, I, L, OP] = ???
```

Comment:
This is why I stopped implementing the "standard" engine. I would have to refactor the whole thing, and I don't have time to do that.